DatadogSessionReplay/Sources/Processor/SnapshotProcessor.swift

```diff
@@ -119,7 +118,7 @@ internal class SnapshotProcessor: SnapshotProcessing {
                 .flatMap { records.append($0) }
             }
         } else {
-            telemetry.error("[SR] Unexpected flow in `Processor`: no previous wireframes and no previous RUM context")
```

Comment:
/question We have been incorrectly loggin errors here?

Reply:
I reverted that change. Will be tackled in a separate PR.